roles/edpm_build_images/tasks/post.yaml

```diff
@@ -12,7 +12,8 @@
     - ironic-python-agent
 
 - name: Retag and push the images with podified-ci-testing tag
-  when: cifmw_repo_setup_promotion == "podified-ci-testing"
+  when: cifmw_repo_setup_promotion
```

Comment:
ah, you decide to use "safe" way, ok :)
Normally I was expecting to use:

```
when: cifmw_repo_setup_promotion and cifmw_repo_setup_promotion.startswith('podified-ci-testing')
```

Reply:
That code is heavy-tied to what we have in repo-setup, so @evallesp's argument of being explicit convinced me here in the end